src/fvOptions/axialFlowTurbineALSource/axialFlowTurbineALSource.C

```diff
@@ -182,29 +182,34 @@ void Foam::fv::axialFlowTurbineALSource::createBlades()
             elementGeometry[j][0][1] = point.y(); // y location of geom point
             elementGeometry[j][0][2] = point.z(); // z location of geom point
 
+            // Set chord reference direction
+            vector chordDirection = azimuthalDirection_;
+
             // Set span directions for AL source
-            scalar spanSign = axis_ & freeStreamDirection_;
-            vector spanDirection = spanSign*verticalDirection_;
+            // Blades start oriented vertically
+            // Use planform normal to figure out span direction
+            vector planformNormal = freeStreamDirection_;
+            vector spanDirection = chordDirection ^ planformNormal;
+            spanDirection /= mag(spanDirection);
+
+            // Rotate span and chord directions according to azimuth
             rotateVector(spanDirection, vector::zero, axis_, azimuthRadians);
             elementGeometry[j][1][0] = spanDirection.x();
             elementGeometry[j][1][1] = spanDirection.y();
             elementGeometry[j][1][2] = spanDirection.z();
-
-            // Set chord length
-            elementGeometry[j][2][0] = chordLength;
-
-            // Set chord reference direction
-            vector chordDirection = azimuthalDirection_;
             rotateVector(chordDirection, vector::zero, axis_, azimuthRadians);
             elementGeometry[j][3][0] = chordDirection.x();
             elementGeometry[j][3][1] = chordDirection.y();
             elementGeometry[j][3][2] = chordDirection.z();
 
+            // Set chord length
+            elementGeometry[j][2][0] = chordLength;
+
             // Set chord mount
             elementGeometry[j][4][0] = chordMount;
 
-            // Set pitch
-            elementGeometry[j][5][0] = -pitch;
+            // Set element pitch or twist
+            elementGeometry[j][5][0] = pitch;
```

Comment:
Need to check this sign change here. Seems like it's correct based on the geometry though.